src/test/java/com/kryptnostic/datastore/edm/DatastoreTests.java

```diff
@@ -265,7 +276,7 @@ public void testAddExistingPropertyTypeToEntityType() {
         dms.addPropertyTypesToEntityType(ENTITY_TYPE.getNamespace(), ENTITY_TYPE.getName(), properties);
     }
     
-    @Test(expected=BadRequestException.class)
+    @Test(expected=IllegalArgumentException.class)
```

Comment:
The one below is a different method. I can change the expected method for this one to try-and-catch though for consistency.